CorsixTH/CppTest/CMakeLists.txt

```diff
@@ -25,15 +25,21 @@ set (TESTING_FILES
     example.cpp
 )
 
-find_package(Catch2 REQUIRED)
+# Ubuntu 16.04 does not have catch and it requires a different
```

Comment:
Better to leave find_package(Catch2 REQUIRED) and add a FindXXX module in CMake for grabbing the header on systems that don't have it installed with the Catch2config.cmake file. I don't want to maintain this in tree.

Reply:
I looked into using a FindXXX module and `ExternalProject_add` to automatically download the deps. This won't work as a find module needs the header at configuration, but external project add runs at build time.

Digging around I discovered we can just use the CMake find package with a small workaround for users who don't have Catch2 in their package manager. When this is merged I'll add it to the wiki (a copy is in `.travis.yml`)